src/popup/components/AppsBrowserHeader.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="apps-browser-header">
+    <div
+      v-if="selectedApp"
+      class="left"
+    >
+      <AccountSelector
+        v-model="accountAddress"
+        avatar-only
+        :options="aeAccountsSelectOptions"
+        @update:model-value="onAccountChange"
+      />
+
+      <BtnIcon
+        class="icon-btn"
+        data-cy="back-arrow"
+        :icon="BackIcon"
+        @click="back"
+      />
+    </div>
+
+    <div class="title">
+      <Truncate
+        v-if="!selectedApp"
+        :str="$t('pages.titles.appsBrowser')"
+        class="text"
+      />
+      <div
+        v-else
+        class="host"
+      >
+        <SecureIcon
+          v-if="isSecure"
+          class="icon"
+        />
+        {{ selectedAppHost }}
+      </div>
+    </div>
+
+    <div class="right">
+      <BtnIcon
+        v-if="selectedApp"
+        :icon="ThreeDotsIcon"
+        @click="openActions"
+      />
+      <BtnClose
+        class="btn-close"
+        @click="close"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  ref,
+  unref,
+} from 'vue';
+import { useStore } from 'vuex';
+import { useRouter } from 'vue-router';
+import { Encoded } from '@aeternity/aepp-sdk';
+import { PROTOCOL_AETERNITY, MODAL_BOWSER_ACTIONS_DAPP, BROWSER_ACTIONS } from '@/constants';
+import {
+  useAccounts,
+  useUi,
+  useModals,
+} from '@/composables';
+import {
+  ROUTE_ACCOUNT,
+  ROUTE_INDEX,
+  ROUTE_MORE,
+} from '@/popup/router/routeNames';
+import AccountSelector from '@/popup/components/AccountSelector.vue';
+import Truncate from '@/popup/components/Truncate.vue';
+import BtnIcon from '@/popup/components/buttons/BtnIcon.vue';
+import BtnClose from '@/popup/components/buttons/BtnClose.vue';
+
+import SecureIcon from '@/icons/secure-lock.svg?vue-component';
+import ThreeDotsIcon from '@/icons/three-dots.svg?vue-component';
+import BackIcon from '@/icons/chevron.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    BtnClose,
+    Truncate,
+    BtnIcon,
+    SecureIcon,
+    AccountSelector,
+  },
+  props: {
+    selectedApp: { type: Object, required: true },
+    iFrame: { type: Object, required: true },
```

Comment:
```suggestion
    iframe: { type: Object, required: true },
```
`iframe` can be considered as a single word.